### PR TITLE
brandon_amam/affiliate_landing

### PR DESCRIPTION
**Pull request by ahkuma**

### CODE REVIEW

File: src/sass/static/affiliates_ib.scss [hunk 1]
```diff
@@ -27,30 +27,222 @@
...
+                    border-color: $COLOR_ORANGE;
+                    margin-bottom: 0;
+
+                    li {
+                        padding: 10px;
+                        margin: 0;
+                        white-space: normal;
+
+                        a {
+                            width: 100%;
```
Comment (kellybinary) [inline src/sass/static/affiliates_ib.scss]: could you add `outline: none` here?

File: src/sass/static/affiliates_ib.scss [hunk 2]
```diff
@@ -27,30 +27,222 @@
...
+                            border-bottom-width: 1px;
+                            border-color: $COLOR_WHITE;
+                        }
+                    }
+                }
+            }
+        }
+    }
+    .faq {
+        background-image: url('../images/pages//affiliates_ib_landing/background/orange.png');
```
Comment (kellybinary) [inline src/sass/static/affiliates_ib.scss]: there's an extra `/` after `/images/pages/`

File: src/templates/static/affiliate_ib_landing.jsx [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+                                            <div className='gr-11 gr-12-m gr-centered'>
+                                                <p className='no-margin text-align-left'>{it.L('Earn increasingly higher payouts with tiered and laddered commission rates that reward you based on the net revenue generated by your referred clients.')}</p>
+                                                <table>
+                                                    <tr>
+                                                        <th>{it.L('Tier')}</th>
+                                                        <th>{it.L('Total net revenue per month (USD)')}</th>
+                                                        <th>{it.L('Commission rates')}</th>
+                                                    </tr>
+                                                    <tr>
+                                                        <td>{it.L('1')}</td>
```
Comment (kellybinary) [inline src/templates/static/affiliate_ib_landing.jsx]: don't need to translate numbers 😄

File: src/templates/static/affiliate_ib_landing.jsx [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+                                            </div>
+                                        </TabContent>
+                                    </TabContentContainer>
+                                </div>
+                            </TabContainer>
+                        </div>
+                    </div>
+                    <div id='ib'>
+                        <div className='gr-12 gr-padding-20 white-bg-color'>
+                            <p className='no-margin'>{it.L(`Earn daily commissions on the trading activities of the clients you refer to the ${it.website_name} MetaTrader 5 (MT5) platform. Your total commissions will be credited into your account by the end of the day.`)}</p>
```
Comment (kellybinary) [inline src/templates/static/affiliate_ib_landing.jsx]: better to use a placeholder to `it.website_name`

Comment (ahkuma) [reply src/templates/static/affiliate_ib_landing.jsx]: pushed

File: src/sass/static/affiliates_ib.scss (hunk 1 above)
Comment (ahkuma) [reply src/sass/static/affiliates_ib.scss]: @kellybinary Can you tell me why add that?

File: scripts/config/pages.js [hunk 5]
```diff
@@ -90,10 +90,9 @@ module.exports = [
     ['open-positions',                          'static/about/job_descriptions',               'full_width', 'Open Positions'],
     ['open-positions/job-details',              'static/about/job_details',                    'full_width', 'Job Details'],
 
-    ['affiliate/signup',                        'static/affiliates/signup',                    'full_width', 'Affiliate'],
+    ['affiliate-ib',                            'static/affiliate_ib_landing',                 'full_width', 'Affiliate and Ib Programme'],
```
Comment (negarn) [inline scripts/config/pages.js]: should it be `Ib` or `IB`?

File: src/templates/static/_layout/footer.jsx [hunk 6]
```diff
@@ -103,8 +103,7 @@ const Footer = () => (
                             <FooterColumn
                                 header={it.L('Partner With Us')}
                                 items={[
-                                    { text: it.L('Affiliate Programme'),     href: it.url_for('affiliate/signup') },
-                                    { text: it.L('IB Programme'),            href: it.url_for('ib-programme/ib-signup') },
+                                    { text: it.L('Affiliate and IB Programme'), href: it.url_for('affiliate-ib') },
```
Comment (negarn) [inline src/templates/static/_layout/footer.jsx]: alignment of `href`

File: src/templates/static/partners/partners.jsx [hunk 7]
```diff
@@ -33,29 +33,12 @@ const Partners = () => (
...
-                    text={it.L('Earn commissions by introducing new customers to [_1].', it.website_name)}
-                />
-
-                <Column
-                    url={it.url_for('ib-programme/ib-signup')}
-                    image='ib'
-                    header={it.L('IB Programme')}
-                    text={it.L('Earn a commission on each client\'s MT5 trading activity – paid out daily.')}
-                />
-
```
Comment (negarn) [inline src/templates/static/partners/partners.jsx]: we're not adding the new link here?

File: src/templates/static/affiliate_ib_landing.jsx [hunk 8]
```diff
@@ -0,0 +1,485 @@
+import React       from 'react';
+import PaymentLogo from '../_common/components/payment_logo.jsx';
+import { Table }   from '../_common/components/elements.jsx';
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing.jsx]: `from` alignment, also alphabetical ordering by path

File: src/templates/static/affiliate_ib_landing.jsx [hunk 9]
```diff
@@ -0,0 +1,485 @@
...
+                                                                { text: it.L('20 – 39.999%') },
+                                                                { text: it.L('1%') },
+                                                            ],
+                                                            [
+                                                                { text: it.L('40 – 59.999%') },
+                                                                { text: it.L('0.75%') },
+                                                            ],
+                                                            [
+                                                                { text: it.L('60 – 79.999%') },
+                                                                { text: it.L('5%') },
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing.jsx]: do we really need translation of numbers?

File: src/templates/static/affiliate_ib_landing.jsx [hunk 10]
```diff
@@ -0,0 +1,485 @@
...
+                                                { text: it.L('Volatility 75 Index') },
+                                                { text: it.L('10.00') },
+                                            ],
+                                            [
+                                                { text: it.L('Volatility 50 Index') },
+                                                { text: it.L('7.50') },
+                                            ],
+                                            [
+                                                { text: it.L('Volatility 25 Index') },
+                                                { text: it.L('3.50') },
```
Comment (negarn) [inline src/templates/static/affiliate_ib_landing.jsx]: same here

File: src/templates/static/partners/partners.jsx (hunk 7 above)
Comment (ahkuma) [reply src/templates/static/partners/partners.jsx]: @negarn They said it is not needed.

**Pull request merged**